asm/battle_anim_80FE840.s

```diff
@@ -727,30 +727,30 @@ sub_80FEE1C: @ 80FEE1C
 	bne _080FEE3E
 _080FEE36:
 	adds r0, r4, 0
-	bl move_anim_task_del
+	bl DestroyAnimVisualTask
 	b _080FEE70
 _080FEE3E:
-	ldr r0, =gAnimationBankTarget
+	ldr r0, =gAnimBankTarget
 	ldrb r0, [r0]
-	bl GetBankIdentity_permutated
+	bl sub_80A8364
```

Comment:
Why was this renamed from an actual name to a sub_?

Reply:
Because the name was wrong. I think it gets a background id but certainly not a bank.